### DISCUSSION TITLE
How do I display header numbers with dots in between?

### DISCUSSION THREAD

**Discussion by mainrs** [typst/typst#4434] [Q&A] [answered]
I read the documentation and I think `set heading(numbering: "1.1")` should be enough. However, the text renders like this:

![image](https://github.com/typst/typst/assets/5113257/e38ecafb-a575-4c36-9d75-02198a1ff8e3)

There is no separation of header levels using `.`. It should be `2.3`

**Comment by frozolotl** [comment, marked as answer]
Can you give a minimal reproducible example? I can't reproduce this with the following source code:
```typ
#set heading(numbering: "1.1")
= A
= B
== C
== D
== Decision Making
```
![rendered](https://github.com/typst/typst/assets/44589151/78fec3cc-79d7-4248-ac3e-d286abb61452)

**Reply by mainrs** [reply]
I accidentally had, buried deep within my content, the line `set heading(numbering: "1")`. Which occurred at a later position than the one in my layout file... :)
